.ci/static-checks.sh

```diff
@@ -110,6 +110,13 @@ Examples:
 EOT
 }
 
+function remove_tmp_files() {
+	rm -f "$url_map" "$invalid_urls" "$md_links" "$submodule_packages" "$all_packages" "${tmp}"
```

Comment:
I think this won't work, since all these variables are local variables

Reply:
Yes. I think you'd need to do something like:

```sh
# List of files to delete on exit
files_to_remove=()

cleanup()
{
    rm -f ${files_to_remove[@]}
}

foo()
{
    files_to_remove+=(foo.log)

    # ...
}

bar()
{
    files_to_remove+=(bar.log)

    # ...
}

main ()
{
    trap cleanup EXIT

    # ...
}

main "$@"
```